### PR TITLE
Cosmos registration token

### PR DESCRIPTION
**Pull request by yuli-ferna**
Issue: https://github.com/XLabs/portal-bridge-ui-issues/issues/21#issue-2038579086

Result: https://github.com/XLabs/portal-bridge-ui-issues/issues/21#issuecomment-2047626535

Thing to clarify:

- The process for this attest will change a bit as it will not be necessary to connect the wallet for chain cosmos cases.
- This implementation does not work in testnet.
- The token creation process can be done when it is from a non-cosmos chain to a cosmos chain, for now due to the way the contract works it is not possible to register a cosmos token to a non-cosmos chain.

### CODE REVIEW

File: src/hooks/useHandleCreateWrapped.tsx [hunk 1]
```diff
@@ -678,13 +681,64 @@ async function sui(
...
+        dispatch(setIsCreating(false));
+      }
+    } catch (e) {
+      console.error(e);
+      enqueueSnackbar(null, {
+        content: <Alert severity="error">{parseError(e)}</Alert>,
+      });
+      dispatch(setIsCreating(false));
+    }
+  }, 3500);
```
Comment (sebastianscatularo) [inline src/hooks/useHandleCreateWrapped.tsx:730]: maybe make it exponential could help reducing the amount of calls

File: src/hooks/useHandleCreateWrapped.tsx [hunk 2]
```diff
@@ -795,6 +849,15 @@ export function useHandleCreateWrapped(
       !!signedVAA
     ) {
       sui(dispatch, enqueueSnackbar, suiWallet, signedVAA, foreignAddress);
+    } else if (isCosmosChain(targetChain as any) && !!signedVAA) {
```
Comment (sebastianscatularo) [inline src/hooks/useHandleCreateWrapped.tsx:852]: not all the cosmos chains should end into it, for instance Sei and Injective do have special cases, we could talk about it

File: src/utils/consts.ts [hunk 3]
```diff
@@ -481,6 +569,28 @@ export const TBTC_ASSET_ADDRESS = THRESHOLD_TBTC_CONTRACTS[CHAIN_ID_ETH].slice(2
 export const THRESHOLD_ARBITER_FEE = 0;
 export const THRESHOLD_NONCE = 0;
 
+export const WORMCHAIN_CONTRACTS_MAINNET = {
+  core: "wormhole1ufs3tlq4umljk0qfe8k5ya0x6hpavn897u2cnf9k0en9jr7qarqqaqfk2j",
+  token_bridge:
+    "wormhole1466nf3zuxpya8q9emxukd7vftaf6h4psr0a07srl5zw74zh84yjq4lyjmh",
+  ibcShimContract:
```
Comment (sebastianscatularo) [inline src/utils/consts.ts:576]: lets keep the same naming here, change ibcShimContract to ibc_shim_contract

**Review by sebastianscatularo — CHANGES_REQUESTED** [review]

File: src/utils/consts.ts (hunk 3 above)
Comment (yuli-ferna) [reply src/utils/consts.ts]: Done!

File: src/hooks/useHandleCreateWrapped.tsx (hunk 2 above)
Comment (yuli-ferna) [reply src/hooks/useHandleCreateWrapped.tsx]: Done!

File: src/hooks/useHandleCreateWrapped.tsx (hunk 1 above)
Comment (yuli-ferna) [reply src/hooks/useHandleCreateWrapped.tsx]: Done!

**Review by sebastianscatularo — APPROVED** [review]

**Pull request merged**